### PR TITLE
[GH-45] add trollface

### LINKED ISSUE #45: Add trollface meme
Hi,

Could you please add the trollface as a meme?
Ive made a branch here but i dont want to sign a CLA: 
https://github.com/aconitumnapellus/mattermost-plugin-memes/tree/add_trollface

The picture i used is this one: 
https://i.kym-cdn.com/entries/icons/original/000/000/091/TrollFace.jpg

Feel free to steal this one commit i made.

### PR DESCRIPTION
**Pull request by aconitumnapellus**
#### Summary
This PR adds the "Trollface" meme.

#### Ticket Link
Fixes #45

### CODE REVIEW

File: server/memelibrary/assets/metadata/trollface.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+example:
+  - Went to school
+  - Didnt learn anything
```
Comment (hanzei) [inline server/memelibrary/assets/metadata/trollface.yaml:3]: These examples are not orrect. Right now the text is split after the first white space so it should be `"Went to school" "Didnt learn anything"`.

Screenshot from 2021-02-19 11-46-47

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for adding more memes :tada: 

Could you please scale down `server/memelibrary/assets/images/trollface.jpg`? It huge right now.

Could you also please remove the unrelated changes in `server/memelibrary/assets.go`?

File: server/memelibrary/assets/metadata/trollface.yaml (hunk 1 above)
Comment (aconitumnapellus) [reply server/memelibrary/assets/metadata/trollface.yaml:3]: Is it possible to test this out without a pattern?
As "make memelibrary && go run server/plugin.go -out aliens.jpg 'memes. memes everywhere" only works with a defined pattern...

Comment (hanzei) [reply server/memelibrary/assets/metadata/trollface.yaml:3]: You can deploy the plugin to your local server

Comment (aconitumnapellus) [reply server/memelibrary/assets/metadata/trollface.yaml:3]: image
Ive added a pattern for testing and have this now.

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Comment by hanzei** [comment]
Tested locally

**Pull request merged**